Newsletters: add 168 (2021-09-29)

harding opened the pull request:
- [x] *Monday* Stack Exchange section @bitschmidty 
- [x] *Monday* `Bitcoin Core #12677` RPC: Add ancestor{count,size,fees} to listunspent output @jnewbery 
- [x] *Monday* `Eclair #1942` Add success probabilities in path finding @dongcarl
- [x] *Tuesday* Update releases/RCs and add topic links @harding

jnewbery commented:
Reviewed everything and pushed a commit for Bitcoin Core 12677. All looks good so far!

Xekyo commented on _posts/en/newsletters/2021-09-29-newsletter.md:
```diff
@@ -0,0 +1,162 @@
+{% comment %}<!-- https://bitcoin.stackexchange.com/search?tab=votes&q=created%3a1m..%20is%3aanswer -->{% endcomment %}
+{% assign bse = "https://bitcoin.stackexchange.com/a/" %}
+
+- [Why TXID has 256 bits when Bitcoin's address security is 160-bit?]({{bse}}109652)
```
```suggestion
- [Why does the TXID have 256 bits when Bitcoin's address security is 160-bit anyway?]({{bse}}109652)
```
I edited the stack exchange post for grammar

Xekyo reviewed (commented):
Thanks for reminding people about BIP322. I found a few typos. I also edited the question with wonky grammar on Stack Exchange and updated the title here.

bitschmidty commented:
@Xekyo Thanks for the feedback, pushed a commit to resolve.

bitschmidty commented on _includes/specials/taproot/en/14-signmessage.md:
```diff
@@ -0,0 +1,64 @@
+easily trade signed messages.
+
+The [generic signmessage][topic generic signmessage] solution we wrote
+about two years in our [bech32 spending support][bech32ss signmessage]
```
s/two years in/two years ago in

bitschmidty commented on _posts/en/newsletters/2021-09-29-newsletter.md:
```diff
@@ -0,0 +1,162 @@
+  that explicitly use authentication tokens (macaroons) that opt-in to
+  interception can be intercepted.
+
+- [BTCPay Server #2842][] adds the ability to disable (and later
```
I think we could drop this commit from the newsletter. I do not feel strongly about it, however.

harding replied:
Yeah, I was on the edge about this, thanks for weighing in!

harding commented:
@bitschmidty I added a FIXME for you pre-merge Wednesday, if you don't mind checking to see if the link for Bitcoin Core 0.20.2 has been uploaded or not.  If it has, please remove the extraneous sentence.  Thanks!

Made edits for all feedback (hopefully), updated releases, added topic links, and reviewed @jnewbery, @dongcarl, and @bitschmidty contributions (great, as always!).  Thanks everyone!

jnewbery commented:
re-reviewed everything and all looks good to me. Thanks everyone!

One final FIXME to address before merge:

https://github.com/bitcoinops/bitcoinops.github.io/pull/654/files#diff-88183ff1f3ec5dcd1e2980fadd796607850858e8c9aa39473a2b5bd4469dcb30R116

bitschmidty commented:
Squashed and merged! Thanks @harding @dongcarl @jnewbery @Xekyo !
